### ISSUE TITLE
Camera support Victure SC220

### ISSUE THREAD

**Issue by the0s** [TheCrypt0/yi-hack-v4#254]
Will this firmware work with Victure Sc220 ? it seems to be a disguised yi camera.

**Comment by timkrins** [comment]
Just for anyone interested - The chipset of the Victure SC220 camera is a Realtek RTS3903N

**Comment by the0s** [comment]
> Just for anyone interested - The chipset of the Victure SC220 camera is a Realtek RTS3903N

Hey TimKrins, 
do you know if this firmware hack will work with the victure?

**Comment by timkrins** [comment]
I don't think so. See issue https://github.com/TheCrypt0/yi-hack-v4/issues/29

**Comment by tarifari** [comment]
> > Just for anyone interested - The chipset of the Victure SC220 camera is a Realtek RTS3903N
> 
> Hey TimKrins, do you know if this firmware hack will work with the victure?

this one https://github.com/cjj25/Yi-RTS3903N-RTSPServer

dont know if you still need it, but rtmp is possible

**Comment by mjpradeepraj** [comment]
> > > Just for anyone interested - The chipset of the Victure SC220 camera is a Realtek RTS3903N
> > 
> > 
> > Hey TimKrins, do you know if this firmware hack will work with the victure?
> 
> this one https://github.com/cjj25/Yi-RTS3903N-RTSPServer
> 
> dont know if you still need it, but rtmp is possible

I tried  https://github.com/cjj25/Yi-RTS3903N-RTSPServer but does not appear to be working. The camera keeps just rebooting. even the backup script has not run.
